src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: Choose how in-scope links open your PWA with Declarative Link Capturing
+subhead: |
+  Declarative Link Capturing is a proposal for a Web Application Manifest property called
+  `"capture_links"` that lets developers determine declaratively what should happen when the browser
+  is asked to navigate to a URL that is within the application's navigation scope, from a context
+  outside of the navigation scope.
+authors:
+  - thomassteiner
+date: 2021-05-18
+description: |
+  Declarative Link Capturing is a proposal for a Web Application Manifest property called
+  "capture_links" that lets developers determine declaratively what should happen when the browser
+  is asked to navigate to a URL that is within the application's navigation scope, from a context
+  outside of the navigation scope.
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/ynRFiUuv3sccX4qUAAPw.jpg
+alt: A hand and several hanging chains, symbolizing links.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - progressive-web-apps
+  - web-app-manifest
+origin_trial:
+  url: https://developer.chrome.com/origintrials/#/view_trial/4285175045443026945
+---
+
+{% Aside %} Declarative Link Capturing is part of the
+[capabilities project](https://web.dev/fugu-status/) and is currently in development. This post will
+be updated as the implementation progresses. {% endAside %}
+
+## What is Declarative Link Capturing? {: #what }
+
+Clicking links on the web can sometimes be a positively surprising experience. For example, clicking
+a [YouTube link](https://www.youtube.com/watch?v=dQw4w9WgXcQ) on a website like the one you read
+this very article on right now on an iOS device that has the
+[YouTube iOS app](https://apps.apple.com/us/app/youtube-watch-listen-stream/id544007664) installed
+results in the link to be opened in the app, not the browser. The same is true for Android and the
+[YouTube Android app](https://play.google.com/store/apps/details?id=com.google.android.youtube&hl=en&gl=US).
```

Comment:
This is a bit of a mouthful. I had to read it twice. I don't think you need this much though. How about:

>For example, clicking a web page link to YouTube on a mobile device opens the YouTube mobile app, if it is installed.

Reply:
Simplified based on your suggestion.